test/c3chartFactory.js

```diff
@@ -37,6 +46,15 @@
                         //.yAxisType("linear") TODO
                     );
                 });
+            },
+            multiY: function (callback) {
+                require(["test/DataFactory", "src/c3chart/Column"], function (DataFactory, Widget) {
+                    callback(new Widget()
+                        .y2(["C2","C3"])
+                        .columns(DataFactory.ND.multiY.columns)
+                        .data(DataFactory.ND.multiY.data)
+                    );
+                });
```

Comment:
@GordonSmith And here is the API call for AM column
https://github.com/hpcc-systems/Visualization/blob/master/test/amchartFactory.js#L48-L65